staffeln/db/sqlalchemy/alembic/scriptpy.mako

```diff
@@ -0,0 +1,18 @@
+"""${message}
```

Comment:
Can you share who renders this mako template?
I guess alembic db manager will do but want to clarify.

Reply:
yep, alembic it is.